<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MejoresTiempos</class>
 <widget class="QDialog" name="MejoresTiempos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Broadway</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabPrincipiante">
    <attribute name="title">
     <string>Facil</string>
    </attribute>
    <widget class="QTableWidget" name="tablePrincipiante">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>221</width>
       <height>181</height>
      </rect>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tiempo</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tabIntermedio">
    <attribute name="title">
     <string>Intermedio</string>
    </attribute>
    <widget class="QTableWidget" name="tableIntermedio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>221</width>
       <height>181</height>
      </rect>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tiempo</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAvanzado">
    <attribute name="title">
     <string>Dificil</string>
    </attribute>
    <widget class="QTableWidget" name="tableAvanzado">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>221</width>
       <height>181</height>
      </rect>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tiempo</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="labelMejoresTiempos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Broadway&quot;;</string>
   </property>
   <property name="text">
    <string>MEJORES TIEMPOS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Broadway</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
